docs/postgres-migration.md

```
@@ -0,0 +1,27 @@
+```yaml
+instance_groups:
+- name: bosh
+  jobs:
+  - name: postgres    # was: postgres-13
+    release: bosh
+```
+
+No property changes are needed — both jobs share the same `postgres.*`
+property namespace. On the first deploy, the `postgres` job detects the
+existing `/var/vcap/store/postgres-13` data directory and runs `pg_upgrade`
+to PostgreSQL 15 automatically.
```

markdownlint-cli2 (0.23.0)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)